docs/products/kafka/kafka-connect/howto/gcp-bigquery-sink-prereq.rst

```diff
@@ -0,0 +1,50 @@
+Configure GCP for a Google BigQuery sink connector
+==================================================
+
+To be able to sink data from Apache Kafka® to Google BigQuery via the dedicated Aiven connector, you need to perform the following steps in the `GCP console <https://console.cloud.google.com/>`_:
+
+* Create a new `Google service account and generate a JSON service key <https://cloud.google.com/docs/authentication/getting-started>`_ 
+* Verify that BigQuery API is enabled
+* Create a `BigQuery dataset <https://cloud.google.com/bigquery/docs/datasets>`_ where the data is going to be stored
```

Comment:
What if the Dataset already exists? I would mention that this is a step for those starting completely from scratch and without existing data

Reply:
Great, changed wording thanks!